rest/snippets/UserConfiguration/UserProfile.yml

```diff
@@ -17,4 +17,8 @@ UserProfile:
     is_child_profile:
       type: boolean
       example: false
-      description: If this flag is set to "true" then the profile will be blocked from viewing adult content
\ No newline at end of file
+      description: If this flag is set to "true" then the profile will be blocked from viewing adult content
+    age_limit:
```

Comment:
a dumb question;
why is the age limit itself associated with user profile?
shouldn't we be matching the profile against something else that has age limit?

Reply:
I wasn’t there for the grooming. @tlangens should know why it was decided to add it per profile.

Reply:
The age limit is compared to the content ratings of content. If content rating age > age limit, then you can't view the content.
is_child_profile (which I'd say is really bad naming) just specifies whether to filter out adult entertainment content, regardless of age. Some people just don't want naughty content to be shown with the regular stuff.

Reply:
Ok, i guess this works out on bigger picture. We'll just have to implement the age_limit checking on the logic wrapping the model.